protocol/avdl/chat1/local.avdl

```diff
@@ -36,6 +36,7 @@ protocol local {
     string mimeType;           // mime type of the object
     Hash encHash;              // hash of ciphertext object 
     bytes key;                 // encryption key
```

Comment:
Now that we have `verifyKey`, it might make sense to call this `encryptionKey` or something?